nbs/01_Tutorials/01_tutorial.ipynb

```diff
@@ -16,7 +16,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Do you think ❌ the emoji would work here?  Not a big deal just wondering!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/921/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Good idea!</p>